lib/beaker-answers/versions/version20.rb

```diff
@@ -85,24 +83,18 @@ def host_answers(host, master, dashboard, options)
 
       console_a[:q_puppet_enterpriseconsole_auth_user] = console_a[:q_puppet_enterpriseconsole_auth_user_email]
 
-      if smtp_password and smtp_username
-        console_a.merge!({
-                           :q_puppet_enterpriseconsole_smtp_password => "'#{smtp_password}'",
-                           :q_puppet_enterpriseconsole_smtp_username => "'#{smtp_username}'",
-                           :q_puppet_enterpriseconsole_smtp_user_auth => 'y'
-                         })
+      if smtp_password && smtp_username
+        console_a.merge!(:q_puppet_enterpriseconsole_smtp_password => "'#{smtp_password}'",
+                         :q_puppet_enterpriseconsole_smtp_username => "'#{smtp_username}'",
+                         :q_puppet_enterpriseconsole_smtp_user_auth => 'y')
       end
 
       answers = agent_a.dup
-      if host == master
-        answers.merge! master_a
-      end
+      answers.merge! master_a if host == master
```

Comment:
I think we called this out; but let's drop this so as not to force this behavior everywhere.

Reply:
https://rubocop.readthedocs.io/en/latest/cops_style/#styleifunlessmodifier disabled